app/export.go

```diff
@@ -62,6 +64,59 @@ var exportablePreferences = map[ComparablePreference]string{{
 }: "EmailInterval",
 }
 
+func (a *App) GetID() string {
+	return a.ID
+}
+func (a *App) SetID(id string) {
+	a.ID = id
+}
+
+func MissingAttachmentInfo(attachment AttachmentImportData) (*LogFormat, *model.AppError) {
```

Comment:
```suggestion
func GetMissingAttachmentInfo(attachment *AttachmentImportData) *LogFormat {
```
Also, modify this function to accept emoji data as well in the params and return LogFormat for that.